packages/fiori/test/samples/ViewSettingsDialog.sample.html

```diff
@@ -29,10 +29,22 @@ <h3>Usage</h3>
 			<ui5-li slot="sortItems">Company</ui5-li>
 			<ui5-li slot="sortItems">Department</ui5-li>
 		</ui5-view-settings-dialog>
+		<script>
+			let vsdResults = document.getElementById("vsdResults");
+			btnOpenDialog1.addEventListener("click", ()=> {
+				vsdResults.innerHTML = "";
+				vsd1.open();
+			});
+
+			vsd1.addEventListener("confirm", function(evt) {
+				vsdResults.innerHTML = JSON.stringify(evt.detail);
+			});
+
+		</script>
 	</xmp></pre>
 	<script>
-		var vsdResults = document.getElementById("vsdResults");
-		btnOpenDialog1.addEventListener("click", function () {
+		let vsdResults = document.getElementById("vsdResults");
```

Comment:
The code that is actually executed should be in ES5, as we don't transpile the samples and it won't work on IE. The code between the "pre" tags (above) is fine to remain in ES6 as this what the readers see. But, here you have to use ES5 (function, var, etc..)

Reply:
Done